i18n/md/collections/2020/en/categories/all.md

```diff
@@ -5,7 +5,7 @@ language: en
 
 2020 is a very special year for many reasons. The most important one: for the first time in 5 years the overall winner of the _Rising Stars_ is not called {vuejs} but {deno}... what a surprise!
 
-#### The Year of the Deno
+#### The Year of Deno
```

Comment:
I guess you might not even need this heading as you moved the Tailwind content to the new category

Reply:
Yes, we don't need it any more.
I wanted to make a joke after having listened to a song called "The Year of the Cat" (for the seventies!) actually 😅

Reply:
Gotcha, haha. I guess I didn't get the joke 😄